src/components/Cluster/ClusterDetail/V5ClusterDetailTable.js

```diff
@@ -369,6 +373,11 @@ class V5ClusterDetailTable extends React.Component {
       !isClusterCreating(cluster) &&
       !isClusterUpdating(cluster);
 
+    const machineTypeLabel =
+      provider === Providers.AWS ? 'Instance type' : 'VM Size';
```

Comment:
This is a little extra. Noticed that on both AWS and Azure this label said `Instance type`, and it should be `VM Size` on Azure